backend/server.js

```diff
@@ -1380,33 +1544,56 @@ async function employeeList(request, response){
     }
         let userList = await new Promise((resolve, reject) => {
             let sql = `SELECT * FROM user WHERE storeId=${request.session.storeId} ORDER BY id`;
-            let a = [0];
-            i = 0;
+            let rv = [];
             
             db.all(sql, [], (err, rows) => {
                 if (err) {
                     reject(err);
                 }
                 rows.forEach((row) => {
-                    b = [ row.id, row.username, row.name,  row.superuser];
-                    a[i] = b;
-                    i++;
+                    valueToAdd = [ row.id, row.username, row.name,  row.superuser];
+                    rv.push(valueToAdd);               
                 });
-                resolve (a);
+                resolve (rv);
             });
         });
```

Comment:
forstår ikke helt hvorfor du vil lave hver row om til en array istedet for at holde det på objekt form sådan du kan bruge de gængse navne som row.id, row.name og så videre nede i resten af koden